chai.js

```diff
@@ -3496,16 +3496,27 @@ module.exports = function (chai, _) {
     if (msg) flag(this, 'message', msg);
```

Comment:
I think the docs above need to specify that this assertion supports the `contains` flag since it ultimately changes its behaviour. It would also be good to add an example.